prometheus/src/test/java/org/opensearch/sql/prometheus/storage/PrometheusStorageFactoryTest.java

```diff
@@ -130,7 +130,7 @@ void testGetStorageEngineWithWrongAuthType() {
         .thenReturn(Collections.emptyList());
     PrometheusStorageFactory prometheusStorageFactory = new PrometheusStorageFactory(settings);
     HashMap<String, String> properties = new HashMap<>();
-    properties.put("prometheus.uri", "https://test.com");
+    properties.put("prometheus.uri", "https://opensearch.org");
```

Comment:
test.com just dropped out of DNS, leading to URI validation failures. Quick fix.